src/YieldDisburser.sol

```diff
@@ -18,7 +18,8 @@ contract YieldDisburser is OwnableUpgradeable {
     uint48 public minimumTimeBetweenClaims;
     mapping(address => uint256[]) public holderToDistribution;
 
-    event BaseYieldDistributed(uint256 amount, address project);
+    event ProjectDistributionStats(uint256 votedYield, uint256 baseYield, uint256 percentage, address project);
```

Comment:
Typically you want your events to be a description of an action, not an entity like "stats" or "total". imo "YieldDistributed" would be a better name for this event. On top of that, I'd consider "percentage" to be redundant data to emit if it can be trivially calculated from the other data emitted. I can see the case for it here because it requires context from the other events to calculate, so I could go either way.

Reply:
So you could theoretically take the total yield claimed for that distribution and then divide it by the voted yield + base yield to derive the percentage, but I think that it would just be great QoL this way, and also for people looking on a block explorer would make more sense.

Reply:
Ok. One thing to keep in mind here is you'e doing integer division, which will also round to the nearest integer. If the frontend is fine with not displaying any precision beyond an integer, then this is fine. Otherwise, I would advocate for off chain calculation of the percentages.

Reply:
> Typically you want your events to be a description of an action, not an entity like "stats" or "total". imo "YieldDistributed" would be a better name for this event. On top of that, I'd consider `percentage` to be redundant data to emit if it can be trivially calculated from the other data emitted. I can see the case for it here because it requires context from the other events to calculate, so I could go either way.

Ok interesting, I think the reason that this is working so far is because here

```
            uint256 votedSplit = ((projectDistributions[i] * halfBalance) / totalVotes);
            breadToken.transfer(breadchainProjects[i], votedSplit + baseSplit);
```
Yield balance times the votes in all our testing has always been higher than the total votes, this is definitely a bug, I'm opening an issue and resolving this convo